src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComAwareWeakReference.NativeAot.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace System
+{
+    internal sealed partial class ComAwareWeakReference
+    {
+        internal static unsafe object? ComWeakRefToObject(IntPtr pComWeakRef, object? context)
+        {
+            return ComWeakRefToComWrappersObject(pComWeakRef, context);
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Why is this here?

Reply:
This is currently present in main, it is not a change from my PR.

I can remove it if you'd like.

Reply:
This is currently present in main, it is not a change from my PR.

I can remove it if you'd like.